spec/models/assign_msa_component_to_service_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'rails_helper'
+
+RSpec.describe AssignSpComponentToServiceEvent, type: :model do
```

Comment:
~~AssignSpComponentToServiceEvent~~ `AssignMsaComponentToServiceEvent`

Reply:
ah well spotted.. will fix it with next PR since Alan is off today

Reply:
https://github.com/alphagov/verify-self-service/pull/276